opencti-platform/opencti-front/src/private/components/vsac/ViewCharts.js

```diff
@@ -149,10 +150,14 @@ class ViewCharts extends Component {
   };
 
   componentDidMount() {
+    this.setState({isDisabled: true});
     const ids = this.state.analysisesIDs.map((i) => i).join();
+    console.log(this.state.analysisesIDs.findIndex( (el) => el === this.state.analysis_id ))
```

Comment:
@ddavisDarkLight can you please remove console logging statements? We should not be checking them as they tend to make their way into production which is not a good thing.